src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchConnectorIT.java

```diff
@@ -102,6 +104,18 @@ public void testHappyPath() throws Exception {
     runSimpleTest(props);
   }
 
+  @Test
+  public void testHappyPathDataStream() throws Exception {
+    isDataStream = true;
+    props.put(DATA_STREAM_TYPE_CONFIG, "logs");
+    props.put(DATA_STREAM_DATASET_CONFIG, "dataset");
+    index = "logs-dataset-" + TOPIC;
+
+    runSimpleTest(props);
+
+    assertEquals(index, helperClient.getDataStream(index).getName());
```

Comment:
we need to `verifySearchResults()` here too

Reply:
I believe `verifySearchResults()` is called in `runSimpleTest()`. Would I need to explicitly call it again?